src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/request/CaseWorkerRoleRequest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.request;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder(builderMethodName = "caseWorkerRoleRequest")
+public class CaseWorkerRoleRequest {
+
+    private String role;
+    private boolean isPrimaryFlag;
+
```

Comment:
this is rework and it has been done as part of tickets 1952/1953 client domain. no need of these beans again

Reply:
this is  need to map the request to the service . i don't have you'r code so i need to build this  otherwise i cannot proceed further